src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -2129,6 +2129,52 @@ def aead_cipher_supported(self, cipher):
             self._lib.EVP_get_cipherbyname(cipher_name) != self._ffi.NULL
         )
 
+    def load_key_and_certificates_from_pkcs12(self, data, password):
+        if password is None:
+            password = self._ffi.NULL
+        elif not isinstance(password, bytes):
+            raise TypeError("Password must be a byte string or None")
+
+        bio = self._bytes_to_bio(data)
+        p12 = self._lib.d2i_PKCS12_bio(bio.bio, self._ffi.NULL)
```

Comment:
Will this error if there's extra crap at the end? Should it? This probably applies to all of our DER loading...